azure-pipelines/build/linux/du/templates/dopapt-docker-steps.yml

```diff
@@ -8,8 +8,16 @@ parameters:
   type: string
 - name: config          # debug/release/minsizerel/relwithdebuginfo
   type: string
+- name: skipTests
```

Comment:
>skipTests [](http://example.com/codeflow?start=8&length=9)

skipTests not required here. Delete it and also the commented lines below.

Reply:
I can't remove it because do-docker-jobs.yml calls the jobs with it, if I remove it, the pipeline fails from the start.